components/eam/src/physics/cam/zm_conv.F90

```diff
@@ -1004,18 +1888,41 @@ subroutine zm_conv_evap(ncol,lchnk, &
     flxprec(:ncol,1) = 0._r8
     flxsnow(:ncol,1) = 0._r8
     evpvint(:ncol)   = 0._r8
+    omsm=0.99999999_r8
```

Comment:
This is a weird number. it deserves a comment explaining what it's for and why the particular precision was chosen.

Reply:
It is a number near unity to prevent problems due to round off error. For consistency with MG2, omsm=0.99999_r8 is used.